capture-traffic/capture-network-traffic.md

```diff
@@ -45,15 +45,12 @@ The **Network Capture** mode is in Beta state and, currently, has some known lim
 
 - The network capturing mode captures IPv4 traffic, while IPv6 is not yet supported.
 
-- The network capturing mode is incompatible with most VPN tools.
+- Some VPN tools might close the VPN connection if a third-party network extension is enabled. You can try to workaround that by making the VPN connection **before** starting Fiddler Everywhere and then enabling its network capturing mode.
```

Comment:
"Some" and "might" both signal uncertainty. Do we need both in this sentence?  And then when we mention the workaround - is that a suggestion when working with VPNs that doesn't close the connection or is it an advice for when you don't want to rely on luck so to say.